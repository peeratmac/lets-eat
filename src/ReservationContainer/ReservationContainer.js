import React from 'react';

const ReservationContainer = () => {
  return (
    <div>
      <h1>Reservation Container</h1>
    </div>
  );
};

export default ReservationContainer;
