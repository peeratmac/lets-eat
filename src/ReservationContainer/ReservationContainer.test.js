import React from 'react';
import { shallow } from 'enzyme';
import ReservationContainer from './ReservationContainer';

describe('RESERVATION CONTAINER', () => {
  it('should match the ReservationContainer Component Snapshot', () => {
    const mockReservations = [
      { name: 'Peerat', date: '12/25', time: '5:30 pm', number: '555' },
      { name: 'Consuelo', date: '10/31', time: '7:45 am', number: '10' }
    ];
    const mockCancelReservation = jest.fn();

    const wrapper = shallow(
      <ReservationContainer
        reservations={mockReservations}
        cancelReservation={mockCancelReservation}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
