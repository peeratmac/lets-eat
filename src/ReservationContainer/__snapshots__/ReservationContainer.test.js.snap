// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RESERVATION CONTAINER should match the ReservationContainer Component Snapshot 1`] = `
<div>
  <ReservationCard
    cancelReservation={[MockFunction]}
    date="12/25"
    key="0"
    name="Peerat"
    number="555"
    time="5:30 pm"
  />
  <ReservationCard
    cancelReservation={[MockFunction]}
    date="10/31"
    key="1"
    name="Consuelo"
    number="10"
    time="7:45 am"
  />
</div>
`;
